obs-to-maven

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/python3
+#!/usr/bin/python2
```

Comment:
Don't change the shebang here: python3 is the primary supported version. Python2 shebang is adjusted in the debian packages.